static/app/views/organizationGroupDetails/groupReplays/groupReplays.tsx

```diff
@@ -0,0 +1,162 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import FeatureBadge from 'sentry/components/featureBadge';
+import Link from 'sentry/components/links/link';
+import PageHeading from 'sentry/components/pageHeading';
+import Pagination from 'sentry/components/pagination';
+import {PanelTable} from 'sentry/components/panels';
+import {IconArrow} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {PageContent, PageHeader} from 'sentry/styles/organization';
+import space from 'sentry/styles/space';
+import {NewQuery} from 'sentry/types';
+import DiscoverQuery from 'sentry/utils/discover/discoverQuery';
+import EventView from 'sentry/utils/discover/eventView';
+import {getQueryParamAsString} from 'sentry/utils/replays/getQueryParamAsString';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import {useParams} from 'sentry/utils/useParams';
+import ReplayTable from 'sentry/views/replays/replayTable';
+import {Replay} from 'sentry/views/replays/types';
+
+const DEFAULT_DISCOVER_LIMIT = 50;
+
+const GroupReplays = () => {
+  const location = useLocation();
+  const organization = useOrganization();
+  const params = useParams();
+
+  const getEventView = () => {
+    const {groupId} = params;
+    const eventQueryParams: NewQuery = {
+      id: '',
+      name: '',
+      version: 2,
+      fields: [
+        'eventID',
+        'project',
+        'timestamp',
+        'url',
+        'user.display',
+        'user.email',
+        'user.id',
+        'user.ip_address',
+        'user.name',
+        'user.username',
+      ],
+      projects: [],
+      orderby: getQueryParamAsString(query.sort) || '-timestamp',
+      query: `issue.id:${groupId}`,
+    };
+
+    return EventView.fromNewQueryWithLocation(eventQueryParams, location);
+  };
+
+  const {query} = location;
+  const {cursor: _cursor, page: _page, ...currentQuery} = query;
+
+  const sort: {
+    field: string;
+  } = {
+    field: getQueryParamAsString(query.sort) || '-timestamp',
+  };
+
+  const arrowDirection = sort.field.startsWith('-') ? 'down' : 'up';
+  const sortArrow = <IconArrow color="gray300" size="xs" direction={arrowDirection} />;
+
+  return (
+    <Fragment>
+      <StyledPageHeader>
+        <HeaderTitle>
+          {t('Replays')} <FeatureBadge type="alpha" />
+        </HeaderTitle>
+      </StyledPageHeader>
```

Comment:
We probably don't need this header in there (none of the other tabs have a header).